examples/models/llama/CMakeLists.txt

```diff
@@ -38,6 +38,7 @@ cmake_dependent_option(
 )
 
 option(EXECUTORCH_BUILD_TORCHAO "Build the torchao kernels" OFF)
+option(EXECUTORCH_BUILD_TORCHAO_MPS "Build the torchao mps kernels" OFF)
```

Comment:
Can we overload TORCHAO instead of doing TORCHAO_MPS

Reply:
Less flags to worry about

Reply:
done!